content/momentum/4/http-api-stats/queues-summary-sizes.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
We don't need to document this endpoint.  The queue sizes can be retrieved from /stats/summary which supersedes this endpoint. 
(we still need to go back in sparkpost circonus agent and get queue sizes from /stats/summary).